docs/release-notes/lke/v1.66.0.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Linode Kubernetes Engine v1.66.0
+date: 2024-03-11
+version: 1.66.0
+---
+
+### Changed
+
+- Rename ConfigMap `kube-system/coredns` to `kube-system/coredns-base`.
+
+### Added
+
+- CoreDNS configuration customization capabilities via the `kube-system/coredns-custom` [ConfigMap](https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/#coredns-configmap-options).
```

Comment:
I suggest to link the newly added LKE-specific guide doc for this feature here:
https://github.com/linode/docs/commit/0710c6d03d52a0b676a4a98de7db402c0c98cc57